modules/NewTicket.js

```diff
@@ -185,8 +189,8 @@ export default class NewTicket extends React.Component {
       })
       return (
         <FormGroup key={'categorySelect' + index}>
-          <ControlLabel>{index == 0 ? '问题分类：' : index + 1 + ' 级分类：'}</ControlLabel>
-          <FormControl componentClass="select" value={selectCategory && selectCategory.id || ''} onChange={(e) => this.handleCategoryChange(e, index)}>
+          <ControlLabel>{index == 0 ? t('category') + ' ' : index + 1 + ' ' + t('thCategory') + ' '}</ControlLabel>
```

Comment:
“x level category” -> “Level x category”

Reply:
fixed in ded73ca